selection/selection.py

```diff
@@ -1,36 +1,31 @@
-from typing import List
-import numpy as np
+from typing import Dict, List
 import dataclasses
 
 
 @dataclasses.dataclass
+#dict {"targets": {"dog":[list of IDs], ...}, "nontargets": [list of IDs]}
 class TrainingSet:
-    target_ids: List[str]
-    nontarget_ids: List[str]
+    targets: Dict[str, List[str]]
+    nontargets: List[str]
 
 
 # include additional dependencies as needed
+from random import sample
 
 
 class TrainingSetSelection:
     def __init__(
         self,
-        target_vectors: np.ndarray,
-        target_ids: List[str],
-        nontarget_vectors: np.ndarray,
-        nontarget_ids: List[str],
+        train_embeddings,
+        train_set_size,
+        audio_flag=False
     ) -> None:
-        """
-        Arguments:
-            target_vectors:
-            target_ids:
-            nontarget_vectors:
-            nontarget_ids:
-        """
```

Comment:
i think we should keep this docstring placeholder, since we need to fill everything out with docstrings before sharing the alpha (fire will also use our main function docstrings for outputting commandline help)